contracts/forwarder/IForwarder.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier:MIT
-pragma solidity ^0.6.2;
-pragma experimental ABIEncoderV2;
+pragma solidity >=0.7.5;
```

Comment:
The rest of the files seem to use ^, not >=
Is there a reason to ever prefer >= ? I understand that it can allow compilation with an incompatible version of solidity, can it?

Reply:
interfaces are included by users (devs) code, so I don't want to require them to be 0.7
so interfaces (and BasePaymaster, BaseRelayRecipient) are >=.